src/lib/navigation/wizard-navigation-allow-forward.spec.ts

```diff
@@ -0,0 +1,178 @@
+import {Component, ViewChild} from '@angular/core';
+import {async, ComponentFixture, fakeAsync, TestBed, tick} from '@angular/core/testing';
+import {ArchwizardModule} from '../archwizard.module';
+import {WizardComponent} from '../components/wizard.component';
+import {checkWizardState} from '../util/test-utils';
+
+@Component({
+  selector: 'aw-test-wizard',
+  template: `
+    <aw-wizard [awNavigationMode] navigateForward="allow">
+      <aw-wizard-step stepTitle='Steptitle 1'>
+        Step 1
+      </aw-wizard-step>
+      <aw-wizard-step stepTitle='Steptitle 2'>
+        Step 2
+      </aw-wizard-step>
+      <aw-wizard-step stepTitle='Steptitle 3'>
+        Step 3
+      </aw-wizard-step>
+    </aw-wizard>
+  `,
```

Comment:
Please remove the trailing `,`

Reply:
@madoar , Sorry, did not notice your comment in time. Will remove the commas right away.

Is it possible to configure Codacy and tell it not to issue warnings about trailing commas?

Reply:
It should be possible. The question is: should we do that? In my opinion a trailing comma suggests that the programmer forgot to add a value/parameter which is normally an indicator for a bug, right?

Reply:
Commas removed in b1cdb91

Reply:
@madoar , From what you are saying, it would be reasonable for Codacy to ban trailing commas, but it is doing the exact opposite (requires trailing commas)...

Reply:
It seems like I misunderstood you. I didn't look at the Codacy results during my review.

Reply:
I've changed the Codacy settings to disallow trailing commas!

Reply:
👍